handshake_messages_test.go

```diff
@@ -162,7 +162,7 @@ func (*clientHelloMsg) Generate(rand *rand.Rand, size int) reflect.Value {
 	m.keyShares = make([]keyShare, rand.Intn(4))
 	for i := range m.keyShares {
 		m.keyShares[i].group = CurveID(rand.Intn(30000))
-		m.keyShares[i].data = randomBytes(rand.Intn(300), rand)
+		m.keyShares[i].data = randomBytes(rand.Intn(300)+1, rand)
```

Comment:
This is OK as it doesn't make sense to have CurveID without any data in ``key_share``